handler/handler_test.go

```diff
@@ -85,10 +87,12 @@ func TestContentPublishedHandler_Handle(t *testing.T) {
 			err := eventHandler.Handle(context.Background(), &config.Config{}, &testEvent)
 
 			var avroBytes []byte
+			var testTimeout = time.Second * 5
 			select {
 			case avroBytes = <-pChannels.Output:
-				fmt.Printf("avroBytes: %v\n", avroBytes)
-			case <-time.After(time.Second * 5):
+				log.Event(ctx, "avro byte sent to producer output", log.INFO)
```

Comment:
And again use `t.Log` instead of `log.Event`:
```suggestion
				t.log("avro byte sent to producer output")
```